src/main/kotlin/apply/ui/admin/selections/SelectionView.kt

```diff
@@ -72,30 +73,31 @@ class SelectionView(
     }
 
     private fun createGrid(applicants: List<ApplicantResponse>): Component {
+        val applicationForms: HashMap<Long, ApplicationForm> = HashMap()
```

Comment:
코틀린은 읽기 전용 컬렉션과 변경 가능한 컬렉션을 구별해 제공합니다. 아래의 코드로 개선해 보면 어떨까요?

```suggestion
        val applicationForms = mutableMapOf<Long, ApplicationForm>()
```